.github/workflows/pr-builder.yml

```
@@ -21,6 +21,16 @@ env:
   PRODUCT_NAME: "ThunderID"
   PRODUCT_NAME_LOWER: "thunderid"
 
+inputs:
```

actionlint (1.7.12)

[error] unexpected key "inputs" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)